packages/runtime-common/realm.ts

```diff
@@ -71,6 +71,7 @@ import { mergeRelationships } from './merge-relationships';
 import type { LoaderType } from 'https://cardstack.com/base/card-api';
 
 export type RealmInfo = {
+  url: string | null;
```

Comment:
Added this so we have something like an "id" to be able to reliably filter realms with functions and structures introduced in this PR

Reply:
reverted this change. using the `realmURL` that already exists on the card in addition to the `realmInfo` symbol/property